claws-mail 3.9.1 reproducible segfaults on exit when compiled with NNTP support.

Apparently the pointer returned by account_get_list() is truncated to an integer due to a missing function definition:

(gdb) r
Starting program: /usr/local/bin/claws-mail 
[New LWP 101084]
[New Thread 80ac06400 (LWP 101084)]

Program received signal SIGSEGV, Segmentation fault.
[Switching to Thread 80ac06400 (LWP 101084)]
0x00000000005331f9 in nntp_disconnect_all (have_connectivity=1) at news.c:1418
1418		for (list = account_get_list(); list != NULL; list = list->next) {
(gdb) where
#0  0x00000000005331f9 in nntp_disconnect_all (have_connectivity=1) at news.c:1418
#1  0x0000000000669cf3 in nntp_main_done (have_connectivity=1) at nntp-thread.c:163
#2  0x00000000004e59aa in exit_claws (mainwin=0x80acf23c0) at main.c:1811
#3  0x00000000004e2627 in main (argc=1, argv=0x7fffffffd8b0) at main.c:1739
(gdb) p list
$1 = (GList *) 0xb836a00


Gentoo bug: https://bugs.gentoo.org/show_bug.cgi?id=469838
Upstream bug: http://www.thewildbeast.co.uk/claws-mail/bugzilla/show_bug.cgi?id=2924
Upstream fix: http://git.claws-mail.org/?p=claws.git;a=patch;h=8891f6e23e51b38c54c7b717fa0922cb101b76a2

--- a/src/news.c
+++ b/src/news.c
@@ -56,6 +56,7 @@
 #include "remotefolder.h"
 #include "alertpanel.h"
 #include "inc.h"
+#include "account.h"
 #ifdef USE_GNUTLS
 #  include "ssl.h"
 #endif
